packages/analytics-js-plugins/src/consentManager/index.ts

```diff
@@ -9,6 +15,30 @@ const ConsentManager = (): ExtensionPlugin => ({
   initialize: (state: ApplicationState) => {
     state.plugins.loadedPlugins.value = [...state.plugins.loadedPlugins.value, pluginName];
   },
+  consentManager: {
+    init(
```

Comment:
Are we going to define more extension points here to determine if a destination is consented or not?

Reply:
No extension point. I can create one utility fn to check that.